Python/Strings/2150.py

```diff
@@ -0,0 +1,24 @@
+# Problema 2150 - Beecrowd - String - Nível 4
+
+# Leitura até o fim do arquivo
+while True:
+    try:
+        # Leitura do alfabeto alienígena
+        alfabeto = input()
+
+        # Leitura da frase
+        frase = input()
+
+        # Iniciando o contador
+        total = 0
+
+        # Contando as ocorrências 
+        for letra in alfabeto:
+            total += frase.count(letra)
```

Comment:
Só por questão didática alterei a variável alfabeto para vogais.

```suggestion
        # Leitura das vogais alienígenas
        vogais = input()

        # Leitura da frase
        frase = input()

        # Iniciando o contador
        total = 0

        # Contando as ocorrências 
        for letra in vogais:
            total += frase.count(letra)
```